src/validationinterface.cpp

```diff
@@ -149,6 +151,9 @@ void UnregisterAllValidationInterfaces()
     if (!g_signals.m_internals) {
         return;
     }
+
+    g_signals.m_internals->Iterate([&](CValidationInterface& callbacks) { callbacks.ValidationInterfaceUnregistering(); });
```

Comment:
👍 Follows the same approach as in `::UnregisterValidationInterface` of being called before the callback gets removed from the list